modules/apps/27-interchain-accounts/controller/keeper/handshake.go

```diff
@@ -42,22 +42,23 @@ func (k Keeper) OnChanOpenInit(
 		err      error
 		metadata icatypes.Metadata
 	)
+
 	if strings.TrimSpace(version) == "" {
-		connection, err := k.channelKeeper.GetConnection(ctx, connectionHops[0])
+		_, err := k.channelKeeper.GetConnection(ctx, connectionHops[0])
 		if err != nil {
 			return "", err
 		}
```

Comment:
If connection is not used any longer, this should just be removed

Reply:
I removed these code

Reply:
I removed it, since the connection is retrieved anyway in `ChanOpenInit`, so if it cannot be found, it will already error there.